python-json-rbac/jwt_keyman.py

```
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
+"""
+Command-line utility for JWT key management operations.
+"""
+import sys
+import argparse
+import json
+import os
+from pathlib import Path
+from typing import Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)